### PR TITLE
Experimental [DNM]

### PR DESCRIPTION
**Pull request by ririchiyo**
This will be merged when more testing is done or Deity says it's finished.

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -677,7 +677,7 @@ datum/mind
 						if (!where)
 							usr << "\red Spawning tome failed!"
```
Comment (ComicIronic) [inline code/datums/mind.dm]: usr and span.

**Comment by d3athrow** [comment]
Need to make another branch to seperate out goofchem and deity cult.

File: code/modules/reagents/newchem/newchem_pyro.dm [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+	sleep(rand(10,30))
+	var/turf/simulated/T = get_turf(holder.my_atom)
+	var/ex_severe = round(created_volume / 10)
+	var/ex_heavy = round(created_volume / 8)
+	var/ex_light = round(created_volume / 6)
+	var/ex_flash = round(created_volume / 4)
+	if(ex_severe > 3)
+		ex_severe = 3
+	if(ex_heavy > 7)
+		ex_severe = 7
```
Comment (clusterfack) [inline code/modules/reagents/newchem/newchem_pyro.dm]: ex_heavy = 7

File: code/modules/reagents/newchem/newchem_pyro.dm [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+	if(ex_light > 14)
+		ex_severe = 14
```
Comment (clusterfack) [inline code/modules/reagents/newchem/newchem_pyro.dm]: ex_light = 14

File: code/modules/reagents/newchem/newchem_pyro.dm (hunk 2 above)
Comment (ComicIronic) [inline code/modules/reagents/newchem/newchem_pyro.dm]: Hell, get rid of the ifs entirely.
They should just be ex_whatever = Clamp(round(created_volume / #), 0, ex_max)

**Comment by Iamgoofball** [comment]
shit, deitycult got merged to this?

**Pull request closed without merging**